.travis.yml

```diff
@@ -2,8 +2,8 @@ language: python
 python:
   - "2.6"
```

Comment:
Perhaps it would be good for hygiene to add:

```
    - pip install flake8
    # flake8 provides code improvement suggestions 
    # the --exit-zero flag prevents flake8 from halting the build
    flake8 . --count --exit-zero --max-line-length=127 --statistics
```